### ISSUE TITLE
TRACE feature records key material

### ISSUE THREAD

**Issue by richsalz** [openssl/openssl#26283]
It looks like the tracing facility can report key material: https://github.com/openssl/openssl/blob/master/crypto/pkcs12/p12_key.c#L111-L128

While the feature is disabled by default, there is no documentation (that I could find) that warns about this.

It is also not clear if a rogue library (as described in #26282) could enable such logging to a target under its own control.

**Comment by t8m** [comment]
I agree secrets should not be printed out in traces. Although tracing can potentially reveal some minor sensitive information, it was IMO not intended to trace out passwords and keys directly.
